lib/sparql_to_sw_solr/instance_title_fields.rb

```diff
@@ -34,8 +34,7 @@ def primary_title_result
               <#{instance_uri}> bf:responsibilityStatement ?o .
               ?ignore ?p ?o
             }
-          }".freeze
-        sparql.query(query)
+          }"
```

Comment:
at this point, this method no longer actually returns results, which is a problem.